Codeinwp/neve
# fix: editor background color [closes #2378]

### Summary
- Fixes issue with editor background not being applied properly when using the Gutenberg plugin.
- Fixes issue with post title placeholder not being visible with dark color scheme;


### Will affect visual aspect of the product

NO

### Test instructions

- Make sure you set a Site Background color, or simply set the dark color palette inside the customizer;
-With and without Gutenberg, the background color should be applied inside the editor;
- When adding a new post, the post title placeholder should be visible when using the dark color scheme;  


- Closes #2378 
- Closes #2407
